lib/net.c

```diff
@@ -25,11 +25,32 @@
 #include "gatekeeper_net.h"
 #include "gatekeeper_config.h"
 
+#define GATEKEEPER_RETA_MAX_SIZE (ETH_RSS_RETA_SIZE_512 / RTE_RETA_GROUP_SIZE)
```

Comment:
Adding a comment where one can learn about "RETA" is important because this information is essential to understand the code later.